core/dispatcher/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -258,7 +277,7 @@ class Invoker(
                     case None => (false, con.run(params, msg.meta, auth.compact, timeoutMillis,
                         action.fullyQualifiedName, msg.activationId.toString)) // cached
                     case Some((start, end, Some((200, _)))) => { // successful init
-                        // Try a little slack for json log driver to process
+                        // TODO:  @perryibm update comment if this is still necessary else remove
```

Comment:
yes, let's keep orthogonal semantic changes to other PRs in case there are issues